sphinx/writers/latex.py

```diff
@@ -552,6 +575,15 @@ def __init__(self, document, builder):
                            builder.config.language)
 
         # set up multilingual module...
+        if self.elements['latex_engine'] == 'pdflatex':
+            if ('T2A' in self.elements['fontenc'] and
+                not self.babel.uses_cyrillic()):
+                self.elements['substitutefont'] = '\\usepackage{substitutefont}'
+                self.elements['sphinxpkgoptions'] += ',cyrnocyr'
```

Comment:
I prefered to move indeed to `sphinx.sty` the fancy latex, rather than have it in template

Reply:
As said above I think now the fancy latex should go into some `sphinxcyrillic.sty`.